### ISSUE TITLE
While creating new Window by passing Id crashes the electron app

### ISSUE THREAD

**Issue by sameer2248** [sentialx/node-window-manager#60]
**Bug description**

When creating new Window by passing Id crashes the electron app
**To Reproduce**

**Expected behavior**

**Screenshots**

<img width="465" alt="Screenshot 2022-05-24 at 4 49 14 PM" src="https://user-images.githubusercontent.com/104753774/170680897-2004d191-6f57-4471-b9b1-bbb16e964d57.png">
<img width="727" alt="Screenshot 2022-05-24 at 4 50 49 PM" src="https://user-images.githubusercontent.com/104753774/170680912-9bd28dde-0581-4665-8e3a-8ab8b86654e8.png">

**Details:**
 - Operating System: Mac os Monterey
 - `node-window-manager` version: 2.2.4
 - Last known working `node-window-manager` version: 2.2.4

**Additional context**

**Comment by pzrq** [comment]
It seems this may be a specific case caused by passing the screen ID of the new Monterey status indicator dot into some TypeScript like:

```
import { Window } from 'node-window-manager';

const demo = (windowId: string) => {
  const parsedWindowId = parseInt(windowId.split(':')[1], 10);
  const window = new Window(parsedWindowId);
  return window.getBounds();
}
// EDIT: It might also be something else, just not sure - the legacy codebase I'm on can be hard to follow along with
```

https://www.macrumors.com/how-to/menu-bar-dot-explanation/

Possibly related #55
